packages/tooling/__tests__/lib/parameters-to-json-schema.test.js

```diff
@@ -848,122 +764,30 @@ describe('defaults', () => {
-        it.each(schemaScenarios)(`scenario: %s`, scenario => {
+        it.each(schemaScenarios)(`scenario: %s`, async scenario => {
           const { requestBody, oas } = fixtures.generateRequestBodyDefaults(refType, scenario, fixtureOptions);
-          expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
+          expect(await parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
         });
       });
     });
-
-    describe('should comply with the `allowEmptyValue` declarative when present', () => {
-      const fixtureOptions = {
-        default: '',
-        allowEmptyValue: true,
-      };
-
-      it.each(schemaScenarios)(
-        'with normal non-$ref, non-inheritance, non-polymorphism cases [scenario: %s]',
-        scenario => {
-          const { requestBody, oas } = fixtures.generateRequestBodyDefaults('simple', scenario, fixtureOptions);
-          expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
-        }
-      );
-
-      it.each(schemaScenarios)('with simple usages of `$ref` [scenario: %s]', scenario => {
-        const { requestBody, oas } = fixtures.generateRequestBodyDefaults('$ref', scenario, fixtureOptions);
-        expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
-      });
-
-      describe.each(polymorphismScenarios)('with usages of `%s`', mod => {
-        it.each(schemaScenarios)(`scenario: %s`, scenario => {
-          const { requestBody, oas } = fixtures.generateRequestBodyDefaults(mod, scenario, fixtureOptions);
-          expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
-        });
-      });
-    });
-
-    describe('should not add a default when one is missing', () => {
-      const fixtureOptions = {
-        default: '',
-        allowEmptyValue: false,
-      };
-
-      it.each(schemaScenarios)(
-        'with normal non-$ref, non-inheritance, non-polymorphism cases [scenario: %s]',
-        scenario => {
-          const { requestBody, oas } = fixtures.generateRequestBodyDefaults('simple', scenario, fixtureOptions);
-          expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
-        }
-      );
-
-      it.each(schemaScenarios)('with simple usages of `$ref` [scenario: %s]', scenario => {
-        const { requestBody, oas } = fixtures.generateRequestBodyDefaults('$ref', scenario, fixtureOptions);
-        expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
-      });
-
-      describe.each(polymorphismScenarios)('with usages of `%s`', mod => {
-        it.each(schemaScenarios)(`scenario: %s`, scenario => {
-          const { requestBody, oas } = fixtures.generateRequestBodyDefaults(mod, scenario, fixtureOptions);
-          expect(parametersToJsonSchema({ requestBody }, oas)).toMatchSnapshot();
-        });
-      });
-    });
-  });
-});
-
-describe('minLength / maxLength', () => {
```

Comment:
See https://github.com/readmeio/oas/pull/141/files?diff=split&w=1#r399093091 for details on why this test case was removed.